torch/distributed/tensor/_api.py

```diff
@@ -282,6 +283,11 @@ def __new__(
         r._local_tensor = local_tensor
         return r
 
+    @torch._disable_dynamo
+    @mark_subclass_constructor_exportable_experimental
+    def __init__(self, *args, **kwargs):
+        pass
```

Comment:
this should probably be `super().__init__()` (otherwise if the default `torch.Tensor` constructor happens to have its own init, we won't be calling it at all)